notebooks/photutils/03_aperture_photometry/03_aperture_photometry.ipynb

```diff
@@ -0,0 +1,970 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "r_in = 3.5  # approximate isophotal extent of inner semimajor axis\n",
+    "r_out = 5.  # approximate isophotal extent of inner semimajor axis\n",
+    "\n",
+    "# Create the apertures\n",
+    "elliptical_annuli = []\n",
+    "for obj in catalog:\n",
+    "    position = (obj.xcentroid.value, obj.ycentroid.value)\n",
+    "    a_in = obj.semimajor_axis_sigma.value * r_in\n",
+    "    a_out = obj.semimajor_axis_sigma.value * r_out\n",
+    "    b_out = obj.semiminor_axis_sigma.value * r_out\n",
+    "    theta = obj.orientation.value\n",
+    "    elliptical_annuli.append(EllipticalAnnulus(position, a_in, a_out, b_out, theta=theta))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set up the figure with subplots\n",
+    "fig, ax1 = plt.subplots(1, 1, figsize=(8, 8))\n",
+    "\n",
+    "# Plot the data\n",
+    "fitsplot = ax1.imshow(np.ma.masked_where(xdf_image.mask, xdf_image_clipped), \n",
+    "                      norm=norm_image)\n",
+    "\n",
+    "# Plot the apertures\n",
+    "for aperture in elliptical_annuli:\n",
+    "    aperture.plot(color='red', alpha=0.4, ax=ax1, fill=True)\n",
+    "for aperture in elliptical_apertures:\n",
+    "    aperture.plot(color='white', alpha=0.7, ax=ax1)\n",
+    "\n",
+    "# Define the colorbar\n",
+    "cbar = plt.colorbar(fitsplot, fraction=0.046, pad=0.04, ticks=LogLocator(subs=range(10)))\n",
+    "labels = ['$10^{-4}$'] + [''] * 8 + ['$10^{-3}$'] + [''] * 8 + ['$10^{-2}$']\n",
+    "cbar.ax.set_yticklabels(labels)\n",
+    "\n",
+    "# Define labels\n",
+    "cbar.set_label(r'Flux Count Rate ({})'.format(xdf_image.unit.to_string('latex')), \n",
+    "               rotation=270, labelpad=30)\n",
+    "ax1.set_xlabel('X (pixels)')\n",
+    "ax1.set_ylabel('Y (pixels)')\n",
+    "ax1.set_title('Elliptical Annuli')\n",
+    "\n",
+    "# Crop to show an inset of the data\n",
+    "ax1.set_xlim(2000, 3000)\n",
+    "ax1.set_ylim(2000, 1000)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Calculating Aperture Corrections\n",
+    "\n",
+    "Now that our apertures have been defined, we can do photometry with them to estimate and account for the background. The aperture correction is calculated by:\n",
+    "- Calculating the count rate within each annulus using `aperture_photometry`\n",
+    "- Dividing each annulus' count rate by each annulus' area to get the mean background value for each annulus\n",
+    "- Taking the mean of those annulus means to get a mean background value for the entire image\n",
+    "- Multiplying the global background mean value times the area of each elliptical photometric aperture, to get the estimated background count rate within each aperture\n",
```

Comment:
`value times the area` -> `value by the area`, to make the sentence read more "Multiplying A by B", instead of "Multiplying A times B", which is slightly redundant. Or perhaps it could be "Calculating A times B", which would be `Multiplying` -> `Calculating`.